backend/src/services/mempool_monitor.py

```
@@ -1,224 +1,299 @@
+                'price': 0,
+                'liquidity': 0,
+                'age_hours': 0
+            }
+
+        if self.socketio:
+            self.socketio.emit('new_token', token_details)
+
+        for callback in self.new_token_callbacks:
+            try:
+                if asyncio.iscoroutinefunction(callback):
+                    asyncio.create_task(callback(token_details))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)